ZenPacks/zenoss/Microsoft/Windows/datasources/ClusterDataSource.py

```diff
@@ -220,7 +207,7 @@ def onSuccess(self, results, config):
                 # specific for cluster disk component
                 data['values'][comp]['state'] = int(state)
                 state = cluster_disk_state_string(int(state))
-            except:
+            except Exception:
                 # handles all other cases
                 data['values'][comp]['state'] = cluster_state_value(state), 'N'
```

Comment:
can data['values'][comp]['state'] be both a single value and a tupple?

Reply:
yes, this is a datapoint and timestamp.  this is how all other datasources are saving datapoints